lib/src/themes/feature_resolver.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:vector_tile_renderer/src/themes/selector.dart';
+
+import '../../vector_tile_renderer.dart';
+
+/// Resolver for resolving the [LayerFeature]s, that a [ThemeLayer] should
```

Comment:
consider removing `that a ThemeLayer should render` since the resolver shouldn't know about rendering or themes.